frontend/src/components/Analysis/AnalysisTable/supportedTasks.tsx

```diff
@@ -0,0 +1,52 @@
+export const condgenTasks = [
```

Comment:
I think it'd probably be better to not call this file `supportedTasks`, as even tasks that are not in this file are supported. What about `taskColumnMapping` or `taskSpecificData`?

Reply:
Separately, it seems that it's not a good idea to separate the definition of `condgenTasks` and `seqlabTasks` into different files. Could they both be defined here?